tools/run_tests/run_xds_tests.py

```diff
@@ -1207,7 +1346,6 @@ def create_target_proxy(gcp, name):
         config = {
             'name': name,
             'url_map': gcp.url_map.url,
-            'validate_for_proxyless': True,
```

Comment:
TODO: should update the configuration check.

Reply:
Not sure I understand this - do we need a TODO in the code or will this be resolved before this PR is submitted?

Reply:
Nvm, the config check of proxyless gRPC blocks us on configuring circuit breakers. I wasn't sure if this is something I need to do or someone else would do it. But it looks Richard is assigned for it. For now, we can just disable the config check.